src/pages/profile/index.js

```diff
@@ -77,6 +89,8 @@ const ViewProfile = () => {
                 className="textarea-small"
                 placeholder="Alex Walker"
                 disabled
+                value={user.firstName + ' ' + user.lastName}
```

Comment:
Please extract this to a utility function - for now in the component.

so this should look like:

```js
value={userDisplayName(user)}
```

And the function can be responsible for how it should display